src/Runner/PhpInterpreter.php

```diff
@@ -28,7 +28,7 @@ class PhpInterpreter
 	private $error;
 
 
-	public function __construct($path, array $args = [])
+	public function __construct($path, $keepSystemIni = FALSE, array $args = [])
```

Comment:
When all arguments are passed low-level in `$args`, the  `-n` should be part of `$args` too, or not?